src/shared/codelens/localLambdaRunner.ts

```diff
@@ -499,14 +499,19 @@ export const invokeLambdaFunction = async (params: {
         templatePath: params.samTemplatePath,
         eventPath,
         environmentVariablePath,
-        debugPort: (params.isDebug) ? params.debugConfig.port.toString() : undefined,
+        debugPort: (params.isDebug && params.debugConfig) ? params.debugConfig.port.toString() : undefined,
         invoker: params.samTaskInvoker
     })
 
     const startInvokeTime = new Date()
     await command.execute()
 
     if (params.isDebug) {
+        if (!params.debugConfig) {
```

Comment:
This validation should go before line 502, where you make use of this validated logic. Leaving it here opens the door to unnecessary code execution. Currently, `command.execute()` would be needless, and any future code added between here and line 502 would be subject to flawed validation.

Reply:
✔️